packages/eslint-plugin/src/rules/no-input-rename.ts

```diff
@@ -10,7 +10,11 @@ import {
   isImportedFrom,
 } from '../utils/utils';
 
-type Options = [];
+type Options = [
+  {
+    allowedRenames: string[];
```

Comment:
I just checked some examples in `@typescript-eslint` repo ([**`no-this-alias`**](https://github.com/typescript-eslint/typescript-eslint/blob/1c1b572c3000d72cfe665b7afbada0ec415e7855/packages/eslint-plugin/src/rules/no-this-alias.ts#L10) and [**`explicit-module-boundary-types`**](https://github.com/typescript-eslint/typescript-eslint/blob/1c1b572c3000d72cfe665b7afbada0ec415e7855/packages/eslint-plugin/src/rules/explicit-module-boundary-types.ts#L19)) and saw that they usually call options like this as "allowedNames".

```suggestion
    readonly allowedNames?: readonly string[];
```

What do you think about this change?

Reply:
Good suggestion, I like it.